examples/medplum-postman-examples/Medplum API.postman_collection.json

```diff
@@ -0,0 +1,1566 @@
+{
+	"info": {
+		"_postman_id": "f859b603-f3c5-4be8-b732-b3e7d668ec4d",
+		"name": "Medplum API",
```

Comment:
Let's have `name` be `Medplum`. Also could rename the file to just `medplum.postman_collections.json`?